src/vs/workbench/services/dialogs/browser/abstractFileDialogService.ts

```diff
@@ -285,7 +286,7 @@ export abstract class AbstractFileDialogService implements IFileDialogService {
 
 			const filter: IFilter = { name: languageName, extensions: distinct(extensions).slice(0, 10).map(e => trim(e, '.')) };
 
-			if (ext && extensions.indexOf(ext) >= 0 && !matchingFilter) {
+			if (!matchingFilter && extensions.indexOf(ext || PLAINTEXT_EXTENSION /* // https://github.com/microsoft/vscode/issues/115860 */) >= 0) {
```

Comment:
Nitpick: you have double commenting here, remove `//` to make it more readable

Reply:
Thanks, pushed a fix.